### ISSUE TITLE
Could not pass event PlayerCastSkillEvent

### ISSUE THREAD

**Issue by yhdev7935** [Sentropic/SkillAPI-s#6]
**Describe the bug**
Could not pass event PlayerCastSkillEvent
I think BlockVector3 of WorldEdit isn't supported in 1.12.2.

**Error Messages**
```
[00:01:50] [Server thread/ERROR]: Could not pass event PlayerCastSkillEvent to SkillAPI vs1.96
java.lang.NoClassDefFoundError: com/sk89q/worldedit/math/BlockVector3
	at com.sucy.skill.hook.WorldGuardHook.asVector(WorldGuardHook.java:50) ~[?:?]
	at com.sucy.skill.hook.WorldGuardHook.getRegionIds(WorldGuardHook.java:41) ~[?:?]
	at com.sucy.skill.listener.AddonListener.onSkillUse(AddonListener.java:93) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor594.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:70) ~[patched_1.12.2.jar:git-Paper-1618]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.12.2.jar:git-Paper-1618]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.12.2.jar:git-Paper-1618]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.12.2.jar:git-Paper-1618]
	at com.sucy.skill.api.player.PlayerData.cast(PlayerData.java:1769) ~[?:?]
	at com.sucy.skill.api.player.PlayerData.cast(PlayerData.java:1738) ~[?:?]
	at com.sucy.skill.api.player.PlayerCombos.applyClick(PlayerCombos.java:138) ~[?:?]
	at com.sucy.skill.listener.ComboListener.onClick(ComboListener.java:84) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor623.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:70) ~[patched_1.12.2.jar:git-Paper-1618]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.12.2.jar:git-Paper-1618]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.12.2.jar:git-Paper-1618]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.12.2.jar:git-Paper-1618]
	at com.sucy.skill.packet.PacketHandler.lambda$callEvent$0(PacketHandler.java:55) ~[?:?]
	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:64) ~[patched_1.12.2.jar:git-Paper-1618]
	at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:423) ~[patched_1.12.2.jar:git-Paper-1618]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:840) ~[patched_1.12.2.jar:git-Paper-1618]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:423) ~[patched_1.12.2.jar:git-Paper-1618]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:774) ~[patched_1.12.2.jar:git-Paper-1618]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:666) ~[patched_1.12.2.jar:git-Paper-1618]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_112]
```

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser chrome
 - Version SkillAPI-s1.96

**Additional context**
Spigot Version: 1.12.2

**Comment by Sentropic** [comment]
Hi. What WorldEdit version are you using?

**Comment by yhdev7935** [comment]
Version 6.1.9;caf0ad9

**Comment by Sentropic** [comment]
Done, will be fixed on 1.97 :)
